multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/DetermineServiceCreateUpdateServiceActionsStep.java

```diff
@@ -133,6 +130,10 @@ private List<ServiceAction> determineActions(ProcessContext context, CloudServic
             actions.add(ServiceAction.UPDATE_PLAN);
         }
 
+        getStepLogger().debug("Service parameters should be updated");
+        getStepLogger().debug("New parameters: " + SecureSerialization.toJson(service.getCredentials()));
+        actions.add(ServiceAction.UPDATE_CREDENTIALS);
+
```

Comment:
I know its an old code, but extract in constants and use placeholders instead of concatenation